hexl/ntt/ntt-internal.hpp

```diff
@@ -17,7 +17,7 @@
 namespace intel {
 namespace hexl {
 
-/// @brief Default C++ NTT implementation of the forward NTT
+/// @brief Radix-4 C++ NTT implementation of the forward NTT
```

Comment:
Is this supposed to be Radix-2? The function name is `ForwardTransformToBitReverseRadix2`.

Reply:
Nice catch